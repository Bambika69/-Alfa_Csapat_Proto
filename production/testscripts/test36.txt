createfield
createvirologist F1
createvirologist F1
createagent virus paralyze V1
createagent virus dance V1
startturn V1
useagent PV1 V1
useagent DV1 V2