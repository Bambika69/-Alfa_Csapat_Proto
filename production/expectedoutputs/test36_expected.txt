Field created: 
ID: F1
Type: Field
Neighbours: null

Virologist created:
ID: V1
Field: F1
Backpack: B1

Virologist created:
ID: V2
Field: F1
Backpack: B2

Agent created:
ID: A1
Type: virus
GeneticCode: paralyze
Virologist: V1
Result: Successful

Agent created:
ID: A2
Type: virus
GeneticCode: dance
Virologist: V1
Result: Successful

V1â€™s turn started.

Agent used on Virologist:
Agent: A1
Virologist: V1
Target: V1
Result: Successful

Agent used on Virologist:
Agent: A2
Virologist: V1
Target: V2
Result: Failed
